src/geos_operations.jl

```diff
@@ -107,41 +126,58 @@ union(obj1::Geometry, obj2::Geometry) = geomFromGEOS(union(obj1.ptr, obj2.ptr))
 #     result
 # end
 
-simplify(obj::Geometry, tol::Real) = geomFromGEOS(simplify(obj.ptr, tol))
-topologyPreserveSimplify(obj::Geometry, tol::Real) =
-    geomFromGEOS(topologyPreserveSimplify(obj.ptr, tol))
-uniquePoints(obj::Geometry) = MultiPoint(uniquePoints(obj.ptr))
-delaunayTriangulationEdges(obj::Geometry, tol::Real = 0.0) =
-    MultiLineString(delaunayTriangulation(obj.ptr, tol, true))
-delaunayTriangulation(obj::Geometry, tol::Real = 0.0) =
-    GeometryCollection(delaunayTriangulation(obj.ptr, tol, false))
-constrainedDelaunayTriangulation(obj::Geometry) =
-    GeometryCollection(constrainedDelaunayTriangulation(obj.ptr))
-
-
-sharedPaths(obj1::LineString, obj2::LineString) =
-    GeometryCollection(sharedPaths(obj1.ptr, obj2.ptr))
+simplify(obj::Geometry, tol::Real, context::GEOSContext = _context) =
+    geomFromGEOS(simplify(obj.ptr, tol, context), context)
+topologyPreserveSimplify(obj::Geometry, tol::Real,
+context::GEOSContext = _context) =
```

Comment:
Consider matching the style of the function signatures in the rest of this file. I.e. for single line declarations, it'll be 
```julia
topologyPreserveSimplify(obj::Geometry, tol::Real, context::GEOSContext = _context) =
```
without any linebreaks.

(If you do this, there are other occurrences in this file.)

Reply:
Good point! I was struggling because I had line wrap on because I didn't want the lines to be too long but I don't think it is worth the lack of readability. I tried to standardize the style a bit more.

Reply:
Thank you!

Maybe we can set this package up with https://github.com/domluna/JuliaFormatter.jl (like in [ArchGDAL.jl](https://github.com/yeesian/ArchGDAL.jl/blob/master/.github/workflows/format_pr.yml)); then we won't have to nitpick on style guides in pull requests and still get e.g. [`margin = 80`](https://github.com/yeesian/ArchGDAL.jl/blob/66066333e7a6c06f3a1086ec1de0b7d4ac3ca79c/.JuliaFormatter.toml#L6) done for us automatically.